hack/dependencies.yml

```diff
@@ -22,18 +22,19 @@
   repo: vmware-tanzu/carvel-kbld
   urlTemplate: https://github.com/vmware-tanzu/carvel-{{.Name}}/releases/download/{{.Version}}/{{.Name}}-{{.OS}}-{{.Arch}}
   version: v0.37.4
+#  To be updated after official kapp release
 - checksums:
     darwin:
-      amd64: e71048d2b11a2c10258079cc134d7d2c2b6584429202a6212306380d3a8c0a30
-      arm64: 3660dd8efe83c1356e05255307fa6f65825ba694d96b93bc38c6a43d7e6d7a8c
+      amd64: 1fee9ff094dfc6750f879df6c111048d20392931d03a41d6c7f68d5db3d05e2d
+      arm64: c2c61530fbcb8812001338eb5c2c41289f810efc5aafb957ff21ac7611d0d9e4
     linux:
-      amd64: b253ea9cf6add07f9497955147dc12e8612c24c36dc9929c9a4fecdc76752bd3
-      arm64: 25491298f6783a8b337d2ebdecf749f7750cf10260fe37086315a9c7da0b558f
+      amd64: f70119163d84b188b818ea7cdf6a88b4482e4e51248d30593a0b47399a1657e6
+      arm64: b1efb38cbc93630e377e74203e64f90b8cdffab2ff2e24c2365baec179a05e3b
   dev: true
   name: kapp
-  repo: vmware-tanzu/carvel-kapp
-  urlTemplate: https://github.com/vmware-tanzu/carvel-{{.Name}}/releases/download/{{.Version}}/{{.Name}}-{{.OS}}-{{.Arch}}
-  version: v0.58.0
+  repo: praveenrewar/kapp
```

Comment:
let's just not forget to change this back before we merge

Reply:
Done